### PR TITLE
updating functions in msr_entry.c

### PR DESCRIPTION
**Pull request by slabasan**

### CODE REVIEW

**Comment by slabasan** [comment]
Kernel has been tested successfully on:
- CentOS 7 Minimal / 3.10.0-514.el7.x86_64 / Intel Xeon E5-2667 v3
- CentOS 7.3.1611 / 3.10.0-514.el7.x86_64 / Intel Xeon 2697 v2
- Ubuntu 18.04.1 LTS / 4.15.0-34-generic x86_64 / Intel Xeon Gold 6126
- CentOS 7 Minimal / 3.10.0-514.el7.x86_64 / Intel Xeon E5-2667 v3

File: msr_entry.c [hunk 1]
```diff
@@ -111,82 +110,103 @@ static ssize_t msr_read(struct file *file, char __user *buf, size_t count, loff_
         return -EACCES;
     }
 
-    for (; count; count -= 8)
+    if (count > 8)
```
[the remark this answers is not in the document]
Comment (slabasan) [reply msr_entry.c]: Thanks for the feedback @mcfadden8!

**Comment by slabasan** [comment]
This PR includes (the title could probably use an update to reflect the broader scope):
- [x] rewrite of `msr_entry.c` functions
- [x] update to new LLNL release number
- [x] update `LICENSE` to LGPL v2.1
- [x] add new files: `RELEASE`, `NOTICE`, and `COPYRIGHT`
- [x] update file headers to use shortened SPDX verbage
- [x] update copyright year to 2020
- [ ] add note about rewritten code based off original msr kernel

**Comment by rountree** [comment]
Looks good.  Thanks for the breakdown.

**Pull request closed without merging**